up simple chinese translation

rsesot opened the pull request:
Sorry to bother you,
I don’t know what your willingness is for localized software. I have uploaded a translation into Simplified Chinese. If you are interested, I can also add a translation into Traditional Chinese.

if author is not interested in localized software. Can you also tell me how to build winlator software? Because git cloned it locally and then opened android studio, I found that it kept prompting the error as shown below. I also tried the prompts and tried it. I felt like it was some files missing.
![截图 2024-07-21 14-56-19](https://github.com/user-attachments/assets/bd126b10-439d-4d1e-bf0b-b1d96ed94052)

Kashinathpat commented:
1. Add this line on line 5 of build.gradle
`
namespace 'com.winlator'
`

2. Don't remove English texts add Chinese translation in `values-zh` or `values-zh-rTW` folder in strings.xml
```
res/
  ├── values/
  ├── values-zh/          # For Simplified Chinese
  └── values-zh-rTW/      # For Traditional Chinese
```

longjunyu2 commented:
The correct way to add localization is to create a resource folder for the corresponding language. https://developer.android.google.cn/guide/topics/resources/localization?hl=zh-cn

rsesot commented:
Thanks, I'll submit the correct localization files right away

rsesot commented:
I've placed the translation in the correct location as instructed.

I apologize, but I'm not very proficient in Android programming or Android Studio. I believe I encountered an error: 'Constant expression required' when compiling the main branch using Winlator. Here are the detailed error messages:
[error.log](https://github.com/user-attachments/files/16398435/error.log)

![截图 2024-07-27 14-06-39](https://github.com/user-attachments/assets/f7941f36-c4af-4b56-a263-bce50c28acfe)

rsesot commented:
I directly cloned the main branch of winlator, but Android Studio consistently failed to recognize it as an Android project. As a workaround, I created a new project using the native C++ template, deleted the app directory to replace it with the original winlator directory, and finally, the project was recognized. I also added the namespace com.winlator on line 5. After this commit, I don't believe there was an error in the original main branch.
![截图 2024-07-27 14-28-36](https://github.com/user-attachments/assets/5d306977-46ba-48ab-a428-4c72d44541a1)

Kashinathpat commented:
Add [these](https://github.com/ewt45/winlator-fork/commit/6e401034e893819b15c9007e3a1f58460614290a) files in the project with its contents.

and also do `git submodule init` and `git submodule update` in project's app folder and then compile the project.

also only add xml files in `values-zh/` which has English texts you want to translate **colors.xml** and **attrs.xml** values will be taken from normal values folder.

rsesot commented:
Thank you, I'm able to compile it normally now, and the translation works out of the box. However, there are too many meaningless commits in this branch, and I'm still working on adding CJK support to this software for WINE+Linux environments. I've found that simply installing fonts doesn't solve the problem. When opening Chinese, Japanese, or Korean software, the windows are filled with '???' or garbled characters, and there's a high chance of garbled text within the software or games.
I'll close this PR and resubmit a final PR when I'm finished.
I'm sorry to trouble you and I hope you have a great day every day.
